keylime/cli/policies.py

```diff
@@ -43,6 +46,13 @@ def process_allowlist(args):
         # Auto-enable IMA (or-bit mask)
         tpm_policy["mask"] = hex(int(tpm_policy["mask"], 0) | (1 << config.IMA_PCR))
 
+        logger.info(
```

Comment:
Based on the text, would this be a `logger.warning?

Reply:
Good catch, updated!